tests/baselines/reference/discriminatedUnionTypes2.errors.txt

```diff
@@ -144,8 +143,6 @@ tests/cases/conformance/types/union/discriminatedUnionTypes2.ts(132,11): error T
         }
         else {
             x.value;  // Error, x is never

```

Comment:
Not an error anymore, it seems

Reply:
This is an expected effect. After reducing away empty intersections, `x` is no longer a union type and therefore we don't do control flow based narrowing. It is similar to what you'd see if `x` just had the type `{ type: number, value: number }`.

Reply:
I meant that the comment is outdated now